core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -308,14 +325,62 @@ class TeamPolicyInternal<Kokkos::Cuda, Properties...>
         m_vector_length(verify_requested_vector_length(vector_length_request)),
         m_team_scratch_size{0, 0},
         m_thread_scratch_size{0, 0},
-        m_chunk_size(32) {
+        m_chunk_size(32),
+        m_tune_team(true),
+        m_tune_vector(false) {
     // Make sure league size is permissible
     if (league_size_ >= int(Impl::cuda_internal_maximum_grid_count()))
       Impl::throw_runtime_exception(
           "Requested too large league_size for TeamPolicy on Cuda execution "
           "space.");
   }
 
+  /** \brief  Specify league size, request team size */
+  TeamPolicyInternal(const execution_space space_, int league_size_,
+                     const Kokkos::AUTO_t& /* team_size_request */,
+                     const Kokkos::AUTO_t& /* vector_length_request */
+                     )
+      : m_space(space_),
+        m_league_size(league_size_),
+        m_team_size(-1),
+        m_vector_length(-1),
+        m_team_scratch_size{0, 0},
+        m_thread_scratch_size{0, 0},
+        m_chunk_size(32),
+        m_tune_team(true),
+        m_tune_vector(true) {
+    // Make sure league size is permissible
+    if (league_size_ >= int(Impl::cuda_internal_maximum_grid_count()))
+      Impl::throw_runtime_exception(
```

Comment:
probably should add additional brackets around this -- so as to clarify the if () { }